src/Persistence.Tests/Yaml/DeserializerValidTests.cs

```diff
@@ -39,7 +39,7 @@ public class DeserializerValidTests : TestBase
         sut.Should().NotBeNull().And.BeOfType<Screen>();
         sut.Name.Should().Be("Screen1");
         sut.TemplateId.Should().Be("http://microsoft.com/appmagic/screen");
-        sut.Children.Should().NotBeNull().And.BeEmpty();
+        sut.Children.Should().BeNull();
```

Comment:
please avoid having null collections, it leads to nullref exceptions

Reply:
we use nullable which make it very clear what can be null

Reply:
If someone tries to get children on control which should never have children, nullref exception is appropriate in my opinion.